contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -120,18 +101,17 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
         uint256 secondaryFeeAmount
     ) external nonReentrant {
         // Bridging tokens within a single chain is not allowed.
-        if (destinationChainID == currentChainID) {
-            revert CannotBridgeTokenWithinSameChain();
-        }
+        require(
+            destinationChainID != currentChainID,
+            "ERC20Bridge: bridging to same chain"
+        );
 
         // Neither the recipient nor the destination bridge can be the zero address.
-        if (recipient == address(0)) {
-            revert InvalidRecipientAddress();
-        }
-
-        if (destinationBridgeAddress == address(0)) {
-            revert InvalidDestinationBridgeAddress();
-        }
+        require(recipient != address(0), "ERC20Bridge: zero recipient address");
```

Comment:
nit: previous zero address checks only state that the address was invalid (instead of zero). May want to make them consistent.

Reply:
set the errors messages for zero checks to have "zero" in them, think it's more descriptive than just "invalid", tried to keep consistent across the contracts.